.github/workflows/build.yml

```
@@ -26,9 +26,15 @@ jobs:
       run: dotnet restore
     - name: Build
       run: dotnet build --configuration Release --no-restore
+    - name: Version
+      run: |
+        MOD_VERSION=$(grep -oP '<Version>\K[^<]+' BlasII.CheatConsole/BlasII.CheatConsole.csproj)
+        echo "MOD_VERSION=$MOD_VERSION" >> $GITHUB_ENV
+        BUILD_DATE=$(date +'%Y%m%d%H%M%S')
+        echo "BUILD_DATE=$BUILD_DATE" >> $GITHUB_ENV
     - name: Upload
       uses: actions/upload-artifact@v3
       with:
-        name: ${{env.MOD_NAME}}
-        path: publish/${{env.MOD_NAME}}/
-        retention-days: 20
\ No newline at end of file
+        name: ${{ env.MOD_NAME }} v.${{ env.MOD_VERSION }}-${{ env.BUILD_DATE }}
```

```suggestion
        name: ${{ env.MOD_NAME }} v${{ env.MOD_VERSION }} ${{ env.BUILD_DATE }}
```
Use spaces between sections, dashes between date